### PR TITLE
Add edit for choreography names

### PR DESCRIPTION
**Pull request by ViliMakinen**

### CODE REVIEW

File: frontend/src/app/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -1,20 +1,52 @@
...
+export class ToolbarComponent implements OnChanges {
   @Input()
   title?: string;
 
   @Input()
   backUrl?: string;
 
-  constructor(public menuService: MenuService, private router: Router) {
+  @Output()
+  changeChoreographyName = new EventEmitter<string>();
```
Comment (RoopeHakulinen) [inline frontend/src/app/toolbar/toolbar.component.ts:24]: **issue:** is the same toolbar used in acrobatics too? 🤔

File: frontend/src/app/toolbar/toolbar.component.scss [hunk 2]
```diff
@@ -1,4 +1,33 @@
-.title {
-  flex: 1 1 auto;
+.title-bar {
+  display: flex;
```
Comment (RoopeHakulinen) [inline frontend/src/app/toolbar/toolbar.component.scss:2]: **suggestion:** there's the flex-layout library also for this purpose

File: frontend/src/app/choreography.service.ts [hunk 3]
```diff
@@ -15,8 +22,32 @@ export class ChoreographyService {
...
+
+  updateChoreography(choreography: Choreography): void {
+    const dialogRef = this.dialog.open(SaveChoreographyDialogComponent, {});
+    dialogRef
+      .afterClosed()
+      .pipe(
+        filter((result) => result === true),
+        switchMap(() => {
+          return this.http.put<Choreography>(
+            `/api/choreographies`,
```
Comment (RoopeHakulinen) [inline frontend/src/app/choreography.service.ts:44]: **suggestion:** add the id in the path even though we don't use it right now as it is the way to do RESTful interfaces

**Review by RoopeHakulinen — APPROVED** [review]

File: frontend/src/app/toolbar/toolbar.component.ts (hunk 1 above)
Comment (ViliMakinen) [reply frontend/src/app/toolbar/toolbar.component.ts:24]: Nice catch! Changed the name to changeTitle and added a boolean input to determine whether the title should be editable or not.

**Pull request merged**